config/locales/metadata/metadata-pt.yml

```diff
@@ -1,9 +1,12 @@
 ---
 pt:
   metadata:
+    refresh:
+      is_processing: A importação da coleção IIIF está em processamento. Recarregue esta página em alguns minutos para ver as obras importadas.
     upload:
       browse: Navegar
       click_to_browse_a_file: Clique para procurar um arquivo...
+      refresh_is_processing: A importação da coleção IIIF está em processamento. Recarregue esta página em alguns minutos para ver as obras importadas.
```

Comment:
It is strange to see this message here when it was not added in other languages.